.github/workflows/ci-tests-data.yml

```diff
@@ -94,7 +94,7 @@ jobs:
         # NOTE: do not include coverage report here, see: https://github.com/nedbat/coveragepy/issues/1003
         run: |
           python -m coverage run --source lightning \
-            -m pytest -v --timeout=60 --durations=60
```

Comment:
You probably don't want to commit this. Or do you not care about all other tests?

Reply:
I am debugging a hanging test.